<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerProduct</class>
 <widget class="QDialog" name="ManagerProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#pushButton {
                   background-color: #0078A8; 
                    color: white;
                    border: 2px solid #0078A8; 
                   border-radius: 5px; 
                   padding: 5px 10px;
                   font-weight: bolder;
                   }
QPushButton#pushButton_3 {
                   background-color: #0078A8; 
                    color: white;
                    border: 2px solid #0078A8; 
                   border-radius: 5px; 
                   padding: 5px 10px;
                  font-weight: bolder;
                   }
QPushButton#Editcommodities {
                   background-color: #0078A8; 
                    color: white;
                    border: 2px solid #0078A8; 
                   border-radius: 5px; 
                   padding: 5px 10px;
                  font-weight: bolder;
                   }
QPushButton#pushButton_2{
                   background-color: #0078A8; 
                    color: white;
                    border: 2px solid #0078A8; 
                   border-radius: 5px; 
                   padding: 5px 10px;
font-weight: bolder;
                   }
QPushButton#pushButton_4{
                   background-color: #0078A8; 
                    color: white;
                    border: 2px solid #0078A8; 
                   border-radius: 5px; 
                   padding: 5px 10px;
font-weight: bolder;
                   }
QPushButton#btnSearch{
                   background-color: #0078A8; 
                    color: white;
                    border: 2px solid #0078A8; 
                   border-radius: 5px; 
                   padding: 5px 10px;
font-weight: bolder;
                   }
QPushButton#btnCloseSearch{
                   background-color: #0078A8; 
                    color: white;
                    border: 2px solid #0078A8; 
                   border-radius: 5px; 
                   padding: 5px 10px;
font-weight: bolder;
                   }
QPushButton#pushButton_5{
                   background-color: #0078A8; 
                    color: white;
                    border: 2px solid #0078A8; 
                   border-radius: 5px; 
                   padding: 5px 10px;
font-weight: bolder;
                   }
QLineEdit#txtSearchID{
background-color: #FFFFFF; /* Đặt màu nền là trắng */
    border-radius: 5px; /* Bo góc */
    padding: 5px; /* Đặt đệm */
    color: #000000; /* Đặt màu văn bản là màu đen */
}

QLineEdit#txtID{
background-color: #FFFFFF; /* Đặt màu nền là trắng */
    border-radius: 5px; /* Bo góc */
    padding: 5px; /* Đặt đệm */
    color: #000000; /* Đặt màu văn bản là màu đen */
}                 
QLineEdit#txtName{

background-color: #FFFFFF; /* Đặt màu nền là trắng */
    border-radius: 5px; /* Bo góc */
    padding: 5px; /* Đặt đệm */
    color: #000000; /* Đặt màu văn bản là màu đen */
}      
QLineEdit#txtAmount{

background-color: #FFFFFF; /* Đặt màu nền là trắng */
    border-radius: 5px; /* Bo góc */
    padding: 5px; /* Đặt đệm */
    color: #000000; /* Đặt màu văn bản là màu đen */

}      
QLineEdit#txtPrice{

background-color: #FFFFFF; /* Đặt màu nền là trắng */
    border-radius: 5px; /* Bo góc */
    padding: 5px; /* Đặt đệm */
    color: #000000; /* Đặt màu văn bản là màu đen */
}    
QLineEdit#txtUpdateAmount{

background-color: #FFFFFF; /* Đặt màu nền là trắng */
    border-radius: 5px; /* Bo góc */
    padding: 5px; /* Đặt đệm */
    color: #000000; /* Đặt màu văn bản là màu đen */
}    
QLineEdit#txtSearch
{

background-color: #FFFFFF; /* Đặt màu nền là trắng */
    border-radius: 5px; /* Bo góc */
    padding: 5px; /* Đặt đệm */
    color: #000000; /* Đặt màu văn bản là màu đen */
}   
QDialog#ManagerProduct
{

   background-color: #87CEFA; /* Đặt màu nền 8472FF là trắng */
    color: #000000; /* Đặt màu văn bản là màu đen */
}  
QTableWidget#tableWidget
{

   background-color: #FFFFFF; /* Đặt màu nền là trắng */
    border-radius: 10px; /* Bo góc */
    padding: 5px; /* Đặt đệm */
    color: #000000; /* Đặt màu văn bản là màu đen */
}  </string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>961</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>-20</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>-6</number>
   </property>
   <property name="text">
    <string>Manager Commodities</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>531</width>
     <height>391</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="title">
    <string>Add Commodities</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  Add</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/hagia/Downloads/qtc++/add-folder.svg</normaloff>../../../Users/hagia/Downloads/qtc++/add-folder.svg</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtID">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>ID of Commodities:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Name of Commodities:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAmount">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Amount of Commodities:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Date of Commodities:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Date Expried of Commodities:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Price of Commodities:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPrice">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Month:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Month:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Editcommodities">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  Edit </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/hagia/Downloads/qtc++/modification.png</normaloff>../../../Users/hagia/Downloads/qtc++/modification.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Delete and Search</string>
   </property>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/hagia/Downloads/qtc++/loupe.png</normaloff>../../../Users/hagia/Downloads/qtc++/loupe.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearch">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>ID of Commodities</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  Delete</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/hagia/Downloads/qtc++/bin.png</normaloff>../../../Users/hagia/Downloads/qtc++/bin.png</iconset>
    </property>
   </widget>
   <zorder>pushButton_4</zorder>
   <zorder>btnSearch</zorder>
   <zorder>txtSearch</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Sort by quantity</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Increasing Amounts</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Decreasing Amounts</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>More ExpDate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Little ExpDate</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Update Commodities</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>ID of Commodities:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Amount of Commodities:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUpdateAmount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearchID">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/hagia/Downloads/qtc++/check-mark.png</normaloff>../../../Users/hagia/Downloads/qtc++/check-mark.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Delete Expried Commodities</string>
   </property>
   <widget class="QRadioButton" name="exp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Expried Commodities</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  Delete All Expried </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Users/hagia/Downloads/qtc++/bin.png</normaloff>../../../Users/hagia/Downloads/qtc++/bin.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
